examples/eva_finetuning/eva_finetuning_multi_gpu.py

```diff
@@ -96,7 +96,8 @@
 )
 
 # EVA initialization
-eva_state_dict = get_eva_state_dict(model, dataloader, peft_config)
+# It is important to set `gather_distributed_inputs=True` here if you use a distributed data sampler.
+eva_state_dict = get_eva_state_dict(model, dataloader, peft_config, gather_distributed_inputs=True)
```

Comment:
```suggestion
eva_state_dict = get_eva_state_dict(model, dataloader, peft_config)
```

As `True` is the default.